requirements/features-executable-from-links.yaml

```diff
@@ -27,22 +27,22 @@ requirements:
     checks:
       - name: Test `hello_world_launch_test.py`
         try:
-          - stdin: launch_test test/launch_testing/examples/hello_world_launch_test.py
+          - stdin: launch_test /opt/ros/iron/share/launch_testing/examples/hello_world_launch_test.py
```

Comment:
The problem with this is that this is only correct when you are running the tests from Debians.  It is not correct for the binaries, or when building from source, or when running on Windows.  I'm honestly not sure what to do here; it is a tricky one to describe across all of the platforms.

Reply:
I did notice exactly that. However, I thought this test cases were only being created for debians (at least those are the only ones that appeared when I searched on all the [issues](https://github.com/osrf/ros2_test_cases/issues?page=5&q=is%3Aissue+Launch+testing)).

Reply:
If you look at the closed ones as well (https://github.com/osrf/ros2_test_cases/issues?page=1&q=is%3Aissue+Launch+testing+is%3Aclosed), you'll see tests for Windows, from source, and from binary as well.

Reply:
Oh I see what you mean for Windows. Then wouldn't it makes sense to kinda leave it as it is? Perhaps add a little more instructions on what path to add on the command, e.g. `launch_test <path_to_share_folder>/launch_testing/examples/hello_world_launch_test.py` or something like that.